paasta_tools/cli/schemas/adhoc_schema.json

```diff
@@ -26,6 +26,12 @@
                 "exclusiveMinimum": true,
                 "default": 1024
             },
+            "gpus": {
```

Comment:
If gpus cannot be used partially, we probably want to enforce that here. This should be an int instead? Also, if we don't want to allow fractions for gpus, we should change taskproc to only accept ints.